docs/blog/v1.md

```diff
@@ -0,0 +1,188 @@
+[MDX Blog](/blog)
+
+# MDX goes stable
+
+It’s been a year and a half since the first MDX commit and a year since MDX was first announced at
+ZEIT Day.  MDX is a radical paradigm shift in how to write immersive content using components.  It’s
+an open, [authorable format](https://johno.com/authorable-format) that makes it _fun_ to write again.
+
+Over the last year we’ve been working on numerous bug fixes, new features, better parsing, and integration
+tests.  Now, we think it’s ready.  **We’re happy to finally release v1!**
+
+## 🎉 What’s new?
+
+Here’s a rough breakdown on what we’ve been working on:
+
+### Parsing
+
+We’ve put a lot of work into improving the parsing of MDX documents.  This not only includes better
+handling of import and export blocks, but being more flexible with JSX blocks.  We won’t get into the
+nitty gritty here, but rest assured we’ve added many more tests and are handling all the edge cases
+we’ve currently encountered (please open an issue if you find a new parse issue 😸).
+
+### remark-mdx
+
+`remark-mdx` is the syntactic extension for MDX in remark.  It provides the parsing functionality for
+MDX as a _remark plugin_.  It’s a bit meta.  With `remark-mdx` you can parse MDX documents
+and perform transformations before writing them back to MDX strings.  This allows tools like prettier
+to use the exact same parser as MDX core by installing the library.
+
+### Custom pragma
+
+With v1 we’ve moved away from using `MDXTag` and are using a custom pragma which wraps `React.createElement`.
+We decided to update this approach so the JSX output is more legible, lighterweight, and more customizable.
+**This means that we can add support for any library that speaks JSX!**
```

Comment:
Is `library that speaks JSX` the correct jargon? I’m not sure what the term library expands to here, React, Vue, etc? The term Library is also previously used as `MDX library` relating to our core v. the format, so I’d try and use a different term here. Should it be framework? Pragma?